### ISSUE TITLE
Form workflow 'Send to URL' - Sending Display name instead of Alias

### ISSUE THREAD

**Issue by JasonMcAndrew** [umbraco/Umbraco.Forms.Issues#1202]
When configuring a workflow for a form of type 'Send form to URL', the payload sent to the URL contains the field display names instead of the Alias.

Here are the field display names for two forms, one English, one French.
![image](https://github.com/umbraco/Umbraco.Forms.Issues/assets/26789785/9f07f520-9d4e-40eb-8ec4-593a47261c05)
![image](https://github.com/umbraco/Umbraco.Forms.Issues/assets/26789785/61c60baa-9644-421c-afaf-af46ae257424)

On both forms First Name and Prénom have the same alias. 
It would make much more sense for the alias to be sent in the payload.

![image](https://github.com/umbraco/Umbraco.Forms.Issues/assets/26789785/c3a818a0-4b1c-4b2c-ac32-d249e296304e)

There is an option to add a manual mapping, which allows you input alias names. When using the default mapping the alias names aren't included in the payload.

![image](https://github.com/umbraco/Umbraco.Forms.Issues/assets/26789785/1182dca5-0482-4871-86f0-bd25c84e2bc8)

Reproduction
------------

### Steps to reproduce

On a new or existing form, Configure workflow > Add Workflow > Send form to URL
- Set workflow name
- Input URL
- Select Method Post
- Use default mapping

### Expected result
I would expect alias names to be sent in the payload instead of display names. Display names are different between forms for each language.

### Actual result
See initial screenshots.

**Comment by AndyButland** [comment]
Agree it would make more sense to use the alias here.  Though we'll likely have to make it a setting or configuration item, as there could be integrations already relying on the values that are currently sent by this workflow.

Could you let us know what version of Forms you are using please?

**Comment by JasonMcAndrew** [comment]
> Agree it would make more sense to use the alias here. Though we'll likely have to make it a setting or configuration item, as there could be integrations already relying on the values that are currently sent by this workflow.
> 
> Could you let us know what version of Forms you are using please?

Hi Andy,

That makes sense, having it as a setting to select when configuring the workflow.

We are currently on version 7.5.4.

**Comment by AndyButland** [comment]
OK - I do think we should introduce this as an option but just to set expectations I'm afraid it won't be something we port back to Forms 7 as that's been "end of life" for some time now.

**Comment by dtng95** [comment]
Hi @JasonMcAndrew,
This issue has been fixed and will be included in the next release.
We introduced a new setting for "Send to URL" workflow, called "Default Element For Fields". If you leave this setting empty, or set it as "Caption", the payload will contain field display name. Otherwise, it will contain alias.

<img width="225" alt="image" src="https://github.com/umbraco/Umbraco.Forms.Issues/assets/145089181/8a2c67ad-79a0-4aa8-bd3c-dd70ce04d426">
